docs/guides/postinstall.md

```diff
@@ -0,0 +1,130 @@
+# Introduction
+
+This page explains how to perform a basic set up after installing Linux on a T2 Mac.
+
+## Do you need to do this?
+
+This guide is mainly relevent in the following cases :-
+
+1. If you have installed Linux using an official ISO, instead of a T2 ISO.
+2. The [Make modules load on early boot](#make-modules-load-on-early-boot) section is relevant for those who wish to encrypt their disk drives using LUKS or some other similar software.
+3. If some functionality related to T2 Macs is broken, then you can consider following this guide.
+
+In rest cases, you probably won't need to follow this guide.
+
+## Installing a kernel for T2 support
+
+Installing a kernel with support for T2 Macs is required in order to get the Keyboard, Trackpad, Touch Bar, Audio, Fan and Wi-Fi working.
+
+Many distro maintainers provide compiled kernels which can be installed on your Linux installation. Following are the links to the repos providing such kernels :-
+
+| Linux Distribution   | Kernel with T2 support |
+| -------------------- | ---------------------- |
+| Arch based distros   | https://github.com/Redecorating/linux-t2-arch |
+| Fedora               | https://github.com/mikeeq/mbp-fedora-kernel |
+| Gentoo               | https://github.com/t2linux/T2-Gentoo-Kernel |
+| Manjaro              | https://github.com/NoaHimesaka1873/manjaro-kernel-t2 |
+| NixOS                | TODO |
```

Comment:
@kekrby is there any separate repo for nixOS kernels?

@NoaHimesaka1873 I guess you maintain xanmod kernels as well. Could you tell me where should I place them in this table.